docs/getting-started/delta-lake-format.md

```diff
@@ -0,0 +1,185 @@
+
+:::info
+Why two separate commands?   
+Because you deleted all data first.   
+
+Remember from part 1 that we either need to define a schema for our downloaded files or we need to execute the download steps separately on the first run.
+The first command only executes the download steps, the second command executes everything but the download steps (regex with negative lookahead).
+:::
+
+## Reading Delta Lake Format with Spark
+
+Checking our results gets more complicated now - we can't just open delta lake format in a text editor like we used to do for CSV files.
+We could now use SQL to query our results, that would be even better. 
+One option is to use a Spark session, i.e. by starting a spark-shell.
+But state-of-the-art is to use notebooks like Jupyter for this.
+One of the most advanced notebooks for Scala code we found is Polynote, see [polynote.org](https://polynote.org/).
+
+We will now start Polynote in a docker container, and an external Metastore (Derby database) in another container to share the catalog between our experiments and the notebook.
+To do so, run the following commands in the projects root directory:
+    
+    docker-compose build
+    mkdir -p data/_metastore
+    docker-compose up
+
+You should now be able to access Polynote at `localhost:8192`. 
+
+:::info Docker on Windows
+You can use Docker Desktop for Windows together with Windows command line or Windows Linux Subsystem (WSL2) for this tutorial. But note that Docker Desktop for Windows needs a license for commercial use
+beginning of 2022.
+
+There is a free alternative for Linux or WSL2 called Podman from Redhat, which has a compatible command line and also the Dockerfiles are compatible, see [podman.io](https://podman.io/).
+Further advantages are that Podman is more lightweight - it doesn't need a service and root privileges to run containers.
+Install podman on WSL2 Ubuntu:
+
+    . /etc/os-release
+    echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+    curl -L "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key" | sudo apt-key add -
+    sudo apt-get update
+    sudo apt-get -y upgrade
+    sudo apt-get -y install podman
+
+Install podman-compose for podman in WSL2:
+
+    sudo apt install python3-pip
+    sudo pip3 install podman-compose
+
+After starting `podman-compose up` in the getting-started folder you should now be able to open Polynote on port localhost:8192, as WSL2 automatically publishes all ports on Windows.
+If the port is not accessible, you can use `wsl hostname -I` on Windows command line to get the IP adress of WSL, and then access Polynote over {ip-address}:8192.
+:::
+
+But when you walk through the prepared notebook "SelectingData", you won't see any tables and data yet. 
```

Comment:
I checked with Timo - Metastore container was not running because of some docker build problem (line-endings of scripts). I made an improvement for this in getting-started repo.